libs/shinkai-node-state/src/v2/mutations/sendMessageToJob/useSendMessageToJob.ts

```diff
@@ -43,31 +43,46 @@ export const useSendMessageToJob = (options?: Options) => {
 
       queryClient.setQueryData(
         queryKey,
-        (old: ChatConversationInfiniteData) => {
-          const newMessages = [
-            generateOptimisticUserMessage(
-              variables.message,
-              (variables.files ?? []).map((file) => ({
-                name: file.name,
-                id: file.name,
-                size: file.size,
-                type: FileTypeSupported.Unknown,
-                preview: URL.createObjectURL(file),
-                file,
-                path: file.name,
-                extension: file.name.split('.').pop() ?? '',
-                mimeType: file.type,
-              })),
-            ),
-          ];
+        (old: ChatConversationInfiniteData | undefined): ChatConversationInfiniteData => {
```

Comment:
are these changes needed here? useSendMessageToJob only runs for existing chats, so there should always be at least two initial messages

Reply:
Yeah right. We have to use this hook only for existent chats so we might need to improve the hook name to make it clear. Adding some comments for now